draft-ietf-ppm-dap.md

```diff
@@ -1996,10 +2005,11 @@ struct {
 } PrepareResp;
 ~~~
 
-Next the Helper decrypts each of its remaining report shares as described in
+The helper processes each of the remaining report shares in turn:
+It decrypts each of its remaining report shares as described in
 {{input-share-decryption}}, then checks input share validity as described in
-{{input-share-validation}}. For any report that was rejected, the Helper sets
-the outbound preparation response to
+{{input-share-validation}}. If these validity checkes fail, the Helper sets
```

Comment:
```suggestion
{{input-share-validation}}. If either decryption or validation fails, the Helper sets
```

"validity checks" implies that only input share validation can fail, but it's also possible for decryption to fail.